src/Serialization/serializers.jl

```diff
@@ -53,6 +53,11 @@ end
 
 function end_dict_node(s::SerializerState)
   write(s.io, "}")
+
+  if s.new_level_entry
+    # makes sure that entries after empty dicts add comma
+    s.new_level_entry = false
+  end
```

Comment:
similar to the case for arrays below, serializing an empty dict somewhere in the tree produced syntactically invalid JSON. this seems to fix it

Reply:
thanks!